src/registrar/management/commands/load_organization_data.py

```diff
@@ -0,0 +1,263 @@
+"""Data migration: Send domain invitations once to existing customers."""
+
+import argparse
+import json
+import logging
+import os
+
+from django.core.management import BaseCommand
+from registrar.management.commands.utility.extra_transition_domain_helper import OrganizationDataLoader
+from registrar.management.commands.utility.terminal_helper import TerminalColors, TerminalHelper
+from registrar.management.commands.utility.transition_domain_arguments import TransitionDomainArguments
+from registrar.models import TransitionDomain, DomainInformation
+from django.core.paginator import Paginator
+from typing import List
+
+from registrar.utility.errors import LoadOrganizationError, LoadOrganizationErrorCodes
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Send domain invitations once to existing customers."
+
+    def add_arguments(self, parser):
+        """Add command line arguments."""
+
+        parser.add_argument(
+            "migration_json_filename",
+            help=("A JSON file that holds the location and filenames" "of all the data files used for migrations"),
+        )
+
+        parser.add_argument("--sep", default="|", help="Delimiter character")
+
+        parser.add_argument("--debug", action=argparse.BooleanOptionalAction)
+
+        parser.add_argument("--directory", default="migrationdata", help="Desired directory")
+
+        # Serves as a domain_additional_filename override
+        parser.add_argument(
+            "--domain_additional_filename",
+            help="Defines the filename for additional domain data",
+        )
+
+        # Serves as a organization_adhoc_filename override
+        parser.add_argument(
+            "--organization_adhoc_filename",
+            help="Defines the filename for domain type adhocs",
+        )
+
+    def handle(self, migration_json_filename, **options):
+        """Process the objects in TransitionDomain."""
+
+        # === Parse JSON file === #
+        json_filepath = os.path.join(options["directory"], migration_json_filename)
+
+        # If a JSON was provided, use its values instead of defaults.
+        with open(json_filepath, "r") as jsonFile:
+            # load JSON object as a dictionary
+            try:
+                data = json.load(jsonFile)
+
+                skipped_fields = ["domain_additional_filename", "organization_adhoc_filename"]
+                # Iterate over the data from the JSON file. Skip any unused values.
+                for key, value in data.items():
+                    if value is None or value.strip() == "":
+                        continue
```

Comment:
**nonblocking comment: ** continue and break should be used only when there is no other logical option as they are less easy to follow when reading code. Instead could you just do the negative here. If value is not None and strippedValue!="" and options not in skipped_field:

Reply:
Sure, not a problem. I did that in a few other places so I may need to change that too. Is this generally good practice in Python? It varies by language